package/pyproject.toml

```diff
@@ -0,0 +1,33 @@
+[build-system]
+requires = ["setuptools~=62.3", "wheel~=0.37.1"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name        = "home-assistant-intents"
+version     = "2022.1.0.dev0"
```

Comment:
@balloob  Not sure if this is what we want to do or not for versioning.

Reply:
yeah that works.